YourEasyRent/DataBase/TelegramRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using MongoDB;
+using MongoDB.Driver;
+using Telegram.Bot;
+using YourEasyRent.DataBase.Interfaces;
+using YourEasyRent.Entities;
+
+namespace YourEasyRent.DataBase
+{
+    public class TelegramRepository : ITelegramRepository
```

Comment:
It's still a ProductRepository. They are named after data that they return, in this case - Products.